src/routes/docs/products/auth/storing-user-preferences/+page.markdoc

```diff
@@ -0,0 +1,402 @@
+    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('<PROJECT_ID>');                 // Your project ID
+
+const databases = new Databases(client);
+
+// Store user preferences in a database
+const promise = databases.createDocument(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    ID.unique(),
+    {
+        userId: '<USER_ID>',
+        theme: {
+            mode: 'dark',
+            primaryColor: '#3498db',
+            fontSize: 'medium'
+        },
+        dashboard: {
+            layout: 'grid',
+            widgets: ['calendar', 'tasks', 'notes'],
+            defaultView: 'week'
+        },
+        notifications: {
+            email: true,
+            push: true,
+            frequency: 'daily'
+        }
+    }
+);
+
+// Retrieve user preferences
+const getUserPrefs = databases.listDocuments(
+    '<DATABASE_ID>',
+    '<COLLECTION_ID>',
+    [
+        Query.equal('userId', '<USER_ID>')
+    ]
+);
+```
+```client-flutter
+import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://<REGION>.cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('<PROJECT_ID>');                 // Your project ID
+
+final databases = Databases(client);
+
+// Store user preferences in a database
+final document = await databases.createDocument(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    documentId: ID.unique(),
+    data: {
+        'userId': '<USER_ID>',
+        'theme': {
+            'mode': 'dark',
+            'primaryColor': '#3498db',
+            'fontSize': 'medium'
+        },
+        'dashboard': {
+            'layout': 'grid',
+            'widgets': ['calendar', 'tasks', 'notes'],
+            'defaultView': 'week'
+        },
+        'notifications': {
+            'email': true,
+            'push': true,
+            'frequency': 'daily'
+        }
+    }
+);
+
+// Retrieve user preferences
+final userPrefs = await databases.listDocuments(
+    databaseId: '<DATABASE_ID>',
+    collectionId: '<COLLECTION_ID>',
+    queries: [
+        Query.equal('userId', '<USER_ID>')
+    ]
+);
+```
+{% /multicode %}
+
+{% arrow_link href="/docs/products/databases" %}
+Learn more about Appwrite Databases
+{% /arrow_link %}
+
+# Choosing the right option {% #choosing-the-right-option %}
+
+This comparison table shows the strengths of each storage option:
+
+| Feature | Browser localStorage | User Preferences | Team Preferences | Appwrite Databases |
+|---------|--------------|------------------|------------------|-------------------|
+| **Storage location** | Client-side | Server-side | Server-side | Server-side |
+| **Persistence** | Device-specific | Cross-device | Team-wide | Cross-device |
+| **Size limit** | 5MB per origin | 64kB | 64kB | Unlimited* |
+| **Complexity** | Simple | Simple | Simple | Complex |
+| **Schema validation** | No | No | No | Yes |
+| **Offline access** | Yes | No | No | No** |
```

Comment:
User and Team preferences can also be locally cached.

Reply:
I think it would be better to do without the comparison table since it's problematic